xmrstak/jconf.cpp

```diff
@@ -109,7 +109,8 @@ xmrstak::coin_selection coins[] = {
 	{ "qrl",             	 {cryptonight_monero_v8, cryptonight_monero, 255u},   {cryptonight_monero_v8, cryptonight_monero, 8u}, nullptr },
 	{ "ryo",                 {cryptonight_heavy, cryptonight_heavy, 0u},          {cryptonight_heavy, cryptonight_heavy, 0u},   nullptr },
 	{ "stellite",            {cryptonight_monero_v8, cryptonight_stellite, 255u}, {cryptonight_monero_v8, cryptonight_monero_v8, 0u}, nullptr },
-	{ "turtlecoin",          {cryptonight_aeon, cryptonight_aeon, 0u},            {cryptonight_aeon, cryptonight_aeon, 0u},     nullptr }
+	{ "turtlecoin",          {cryptonight_aeon, cryptonight_aeon, 0u},            {cryptonight_aeon, cryptonight_aeon, 0u},     nullptr },
+    { "zelerius",            {cryptonight_monero_v8, cryptonight_monero, 6u},     {cryptonight_monero_v8, cryptonight_monero, 6u}, "pool.zelerius.com:3333" }
```

Comment:
Could you set the dev pool to `{ "zelerius",            {cryptonight_monero_v8, cryptonight_monero, 6u},     {cryptonight_monero_v8, cryptonight_monero_v8, 0u}, "pool.zelerius.com:3333" }`

Reply:
Yes, changed.